tests/test_dialects.py

```diff
@@ -297,25 +297,25 @@ def test_presto(self):
         )
         self.validate(
             "CAST(a AS ARRAY(INT))",
-            "CAST(a AS ARRAY(INTEGER))",
+            "CAST(a AS ARRAY<INTEGER>)",
             read="presto",
             write="presto",
         )
         self.validate(
             "CAST(ARRAY[1, 2] AS ARRAY(BIGINT))",
-            "CAST(ARRAY[1, 2] AS ARRAY(BIGINT))",
+            "CAST(ARRAY[1, 2] AS ARRAY<BIGINT>)",
             read="presto",
             write="presto",
         )
         self.validate(
-            "CAST(MAP(ARRAY[1], ARRAY[1]) AS MAP(ARRAY(INT(9))))",
-            "CAST(MAP(ARRAY[1], ARRAY[1]) AS MAP(ARRAY(INTEGER(9))))",
+            "CAST(MAP(ARRAY[1], ARRAY[1]) AS MAP(INT,INT))",
```

Comment:
why was this test changed

Reply:
`CAST(MAP(ARRAY[1], ARRAY[1]) AS MAP(ARRAY(INT(9))))` is not a valid Presto query (I think?), so I updated it. But maybe you were trying to test double nested structure like `MAP(ARRAY(INT(9)))`

Reply:
Makes sense, added another test with double nested structure.